packages/expo-dev-launcher/ios/EXDevLauncherAuth.m

```diff
@@ -100,7 +100,7 @@ - (BOOL)initializeWebBrowserWithResolver:(RCTPromiseResolveBlock)resolve andReje
   NSArray<NSDictionary*> *urlTypes = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleURLTypes"];
   
   if (urlTypes != nil) {
-    for (int i = 1; i <= urlTypes.count; i++) {
+    for (int i = 0; i <= urlTypes.count; i++) {
```

Comment:
```suggestion
    for (int i = 0; i < urlTypes.count; i++) {
```
I don't have any context on why this was written the way it was, but it seems like without this additional change you're going to run into out of bounds errors?

Reply:
good catch! thank you :)